spec/unit/resource_spec.rb

```diff
@@ -232,7 +232,7 @@ def config(options = {})
       let(:resource) { namespace.register(Post) }
       let(:post) { double }
       before do
-        allow(Post).to receive(:find_by_id).with('12345') { post }
+        allow(Post).to receive(:find_by).with('id' => '12345') { post }
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.